src/oscar/templates/oscar/catalogue/detail.html

```diff
@@ -28,7 +28,7 @@
             <a href="{{ c.get_absolute_url }}">{{ c.name }}</a>
         </li>
         {% endfor %}
-        <li class="active">{{ product.title }}</li>
+        <li class="active">{{ product.get_title }} {% if product.is_child and product.attribute_summary %}({{ product.attribute_summary }}){% endif %}</li>
```

Comment:
As noted in the previous PR, this will become very long for any product that has more than 2-3 attributes defined (which can happen quite easily). I don't think it is a good default. Same for the product title/heading.

I think the attribute summary needs to be displayed separately from the title. The child product title can be edited separately anyway, so projects can control what is displayed there.

Reply:
> I think the attribute summary needs to be displayed separately from the title. The child product title can be edited separately anyway, so projects can control what is displayed there.

I think we can display attribute summary under the title, so that we can easily identify what variant is it. What do you think?

Reply:
Still don't think this is a good idea. Just try creating a product with 6 or 7 attributes, and this will not render well at all.